CMakeLists.txt

```diff
@@ -44,11 +44,25 @@ else()
     set(SFML_BUILD_EXAMPLES FALSE)
 endif()
 
+# add options to select which modules to build
+sfml_set_option(SFML_BUILD_SYSTEM TRUE BOOL "TRUE to build SFML's System module. This setting is ignored, if any of the other modules is built.")
+if(SFML_OS_WINDOWS OR SFML_OS_ANDROID OR SFML_OS_IOS)
+    sfml_set_option(SFML_BUILD_MAIN TRUE BOOL "TRUE to build SFML's main module (entry point handling).")
```

Comment:
I don't think we need this option either. On iOS and Android it is mandatory, and on Windows it's almost always wanted. And it's just a single C++ file compiled into a static library.